libs/components/src/dialog/dialog/dialog.component.ts

```diff
@@ -9,7 +9,7 @@ export class DialogComponent {
   @Input() dialogSize: "small" | "default" | "large" = "default";
 
   private _disablePadding: boolean;
```

Comment:
Change this to 
```suggestion
  private _disablePadding = false;
```

same with _inline below